app/controllers/concerns/spotlight/search_helper.rb

```diff
@@ -4,9 +4,10 @@ module Spotlight
   # ...
   module SearchHelper
     def search_service(user_params = respond_to?(:search_state, true) ? search_state.to_h : {})
+      search_state = Blacklight::SearchState.new(user_params, blacklight_config)
```

Comment:
Can't you just grab the search_state from the view context?  Why do you have to initialize it yourself?

Reply:
That change alone isn't enough. Spotlight is relying on the behavior in that `search_service` method elsewhere. And it's tangled up with how search state is being managed to be aware of the current exhibit scope. More work should be done to clean this up, but it's beyond the scope of what I can do right now.